### PR TITLE
Custom Metrics API: don't fail requests by selector if metrics for some pods were not found.

### PR DESCRIPTION
**Pull request by kawych**

### CODE REVIEW

File: custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func (t *Translator) metricsFor(values map[string]resource.Quantity, groupResour
 
 	for _, item := range podList.Items {
 		if _, ok := values[fmt.Sprintf("%s", item.GetUID())]; !ok {
-			return nil, provider.NewMetricNotFoundForError(groupResource, metricName, item.GetName())
+			continue
```
Comment (fgrzadkowski) [inline custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go]: How about writing some logs in high verbosity level? I think it'll make debugging easier. WDYT?

Comment (kawych) [reply custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go]: Done.

File: custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go [hunk 2]
```diff
@@ -227,7 +227,8 @@ func (t *Translator) metricsFor(values map[string]resource.Quantity, groupResour
 
 	for _, item := range podList.Items {
 		if _, ok := values[fmt.Sprintf("%s", item.GetUID())]; !ok {
-			return nil, provider.NewMetricNotFoundForError(groupResource, metricName, item.GetName())
+			glog.V(4).Infof("Metric not '%s' not found for pod '%s'", metricName, item.Name)
```
Comment (fgrzadkowski) [inline custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go]: Something wrong with this log message. I think you have extra "not" before metric name.

Comment (kawych) [reply custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go]: Right, fixed

**Pull request merged**